src/components/InventoryGroupDetail/GroupDetailHeader.js

```diff
@@ -62,6 +88,27 @@ const GroupDetailHeader = ({ groupId }) => {
     return groupId;
   };
 
+  let dropdownItems = [
+    <DropdownItem key="rename-group" onClick={() => setRenameModalOpen(true)}>
+      Rename
+    </DropdownItem>,
+    <DropdownItem key="delete-group" onClick={() => setDeleteModalOpen(true)}>
+      Delete
+    </DropdownItem>,
+  ];
+
+  if (isEdgeParityGroupsEnabled) {
+    dropdownItems.push(
+      <DropdownItem
+        key="update-edge-devices"
+        onClick={() => setEdgeUpdateModal({ deviceData: [], isOpen: true })}
+        isDisabled={!edgeDeviceUpdateInfo?.update_valid}
```

Comment:
Just curious if we are supposed to add some tooltip helping users to understand why the option is disabled? Feel free to ignore this comment if UX didn't explicitly ask for it :-)

Reply:
The  UX didn't explicitly ask for it